docs/building.md

```diff
@@ -78,7 +78,12 @@ You can generate the known metadata, but you'll need the `stack` tool installed,
 [The Haskell Tool Stack](https://docs.haskellstack.org/en/stable/README/) tutorial for instructions on how to obtain it.
 
 ```bash
-make metadata --output=metadata.json
+make metadata \
+    frozen_token_symbol=frozen_token \
+    frozen_token_name="Frozen Token" \
+    frozen_token_decimals=0 \
```

Comment:
Does it makes sense to leave this configurable?

Reply:
@sras Ideally the decimal would be the same as the governance token. AFAIU decimal is useful for UI rendering even though we don't really use it in contract logic. So I think it is fine leave it as configurable. @pasqu4le Hope to get your view on this as well.

Reply:
> @sras Ideally the decimal would be the same as the governance token.

Wait, do we have that kind of logic in our contract? So if the governance token have decimals of `2`, does our contract requires freezing 100 governance tokens to put down one vote?

I am not sure what the expectation of such a participant. So say I have 5 governance tokens. In my wallet, it is shown as `5.00`, and suppose I want to vote in a DAO proposal. So call `freeze` on the BaseDAO, and aftewords, my wallet shows the balance of my governance token as `4.99`.  But in the same wallet, in my BaseDAO account, the balance will be shown as `1.00`, (because the decimals is 2 in baseDAO as well), Right?

Does that make sense? Or am I wrong somewhere?

Reply:
>  So say I have 5 governance tokens. In my wallet, it is shown as 5.00, and suppose I want to vote in a DAO proposal. 

AFAIU, it will be shown as `0.05` for governance token if decimal is purely for rendering purpose. So in BaseDAO it will be shown as `0.01`. Not sure if I explain this clearly.

Reply:
>AFAIU, it will be shown as 0.05 for governance token

Yes, I mean, 5 tokens, as displayed by the wallet. So it would be a value of `500` in the contact storage (if the decimals is 2)

Reply:
I agree with @sras, we should probably leave this at `0` and remove the option here from the makefile.

While the translation would work, there is still the problem that the `docs` are based on there being no decimals, for example, from the spec:
> Each vote stakes one frozen token

and
> - The voter MUST have frozen tokens equal to `vote_amount` or more (1 unstaked frozen token is needed for 1 vote) from past `stage`s.

Is "one frozen token" here equal to `1` or `1.0` (aka `100`) tokens? I think this might be confusing for people reading.

`homebase` as well is built with unitary tokens in mind AFAIK, so that would cause even more trouble IMO.

Reply:
Got this, I've made this change now.